tendrl/node_agent/manager/manager.py

```diff
@@ -1,18 +1,20 @@
+import json
 import logging
 import signal
 import socket
+import time
 
 import gevent.event
 import gevent.greenlet
-import json
 import pull_hardware_inventory
-from rpc import EtcdThread
-from tendrl.common.log import setup_logging
 
-from tendrl.node_agent.config import TendrlConfig
+from tendrl.common.log import setup_logging
+from tendrl.common.manager.manager import Manager
+from tendrl.common.manager.manager import SyncStateThread
+from tendrl.common.config import TendrlConfig
 from tendrl.node_agent.persistence.tendrl_definitions import TendrlDefinitions
 
-config = TendrlConfig()
+config = TendrlConfig("node_agent", "/etc/tendrl/tendrl.conf")
```

Comment:
The python project name "node_agent" and the config file path should not be hardcoded